components/server/src/workspace/workspace-service.ts

```diff
@@ -233,10 +233,10 @@ export class WorkspaceService {
     }
 
     async getWorkspace(userId: string, workspaceId: string): Promise<WorkspaceInfo> {
-        const workspace = await this.doGetWorkspace(userId, workspaceId);
-
-        const latestInstancePromise = this.db.findCurrentInstance(workspaceId);
-        const latestInstance = await latestInstancePromise;
+        const [workspace, latestInstance] = await Promise.all([
```

Comment:
Great catch! I reverted the changes as I think it makes sense to do as it was because of the permission checks. [`a1fc587` (#20437)](https://github.com/gitpod-io/gitpod/pull/20437/commits/a1fc587c6983345bc685880f09f0351bb98e7236)